packages/sil-governance/src/routes/dashboard/components/Proposal/VotingPanel/ParticipationData.tsx

```diff
@@ -10,7 +10,7 @@ interface Props {
 }
 
 const ParticipationData = ({ quorum, threshold, tally }: Props): JSX.Element => {
-  const participation = (tally.totalVotes / tally.maxVotes) * 100;
+  const participation = Math.round((tally.totalVotes / tally.maxVotes) * 100 * 100) / 100;
```

Comment:
What about `participation.toFixed(2)` instead?

Reply:
That works too but doesn't avoid trailing 0s. `78.4` would be converted to `78.40` and `78` to `78.00`

Reply:
👍 thanks for checking that.

In this case, can we have a `round(value: number, digits: number): number` function in ui-logic that implements the same logic you use here? This is such a common use case and makes code easier to read.